backend/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 
 app.jwtSecret=jwtsecret
 app.jwtExpirationInMs=604800000
-spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/exdelivery
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3360/exdelivery
```

Comment:
Was this a mistake? If the port is changed here it must be changed in the Dockerfile and docker-compose files as well.

Reply:
No, will alter them then